### PR TITLE
Updated UI of order detail template #7611

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: templates/webshop/sale.jinja [hunk 1]
```diff
@@ -45,65 +45,84 @@
...
+                  <tr>
+                    <td class="col-md-2 col-xs-12">
+                      <img src="{{ CDN }}{{ line.product.default_image.transform_command().thumbnail(200, 200, 'a') }}"
+                      alt="{{ line.product.name }}"
+                      title="{{ line.product.name }}"
+                      class="img-responsive img-rounded margin-auto">
+                    </td>
+                    <td class="col-md-4">
+                      {% if line.product and line.product.displayed_on_eshop %}
+                      <a href="{{ url_for('product.product.render', uri=line.product.uri) }}"><strong>{{ line.description }}</strong></a>
```
Comment (tarunbhardwaj) [inline templates/webshop/sale.jinja]: product.get_absolute_url()

File: templates/webshop/sale.jinja [hunk 2]
```diff
@@ -45,65 +45,84 @@
...
+                      <br>
+                      Qty: {{ line.quantity|int }} {{ line.unit.name }}
+                      {% else %}
+                      {{ line.description }}
```
Comment (tarunbhardwaj) [inline templates/webshop/sale.jinja]: strong?

File: templates/webshop/sale.jinja [hunk 3]
```diff
@@ -45,65 +45,84 @@
...
+                      <br>
```
Comment (tarunbhardwaj) [inline templates/webshop/sale.jinja]: this is common can be pulled out of `if`

**Comment by gautampanday** [comment]
order-page

**Comment by gautampanday** [comment]
payment-webshop

File: templates/webshop/sale.jinja [hunk 4]
```diff
@@ -45,65 +45,82 @@
...
             </div>
           </div>
-
+          <div class="row-fluid">
+            <div class="col-md-6">
+              <div class="panel panel-default order-payment-panel margin-top-30">
+                <div class="panel-heading">
+                  <h4>Payment Method</h4>
+                </div>
+                {% for payment in sale.payments %}
```
Comment (tarunbhardwaj) [inline templates/webshop/sale.jinja]: IMO loop should be on `tr` not on whole table

**Pull request merged**